src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/ToolRestoreUtility.cs

```diff
@@ -52,16 +52,27 @@ public static PackageSpec GetSpec(string projectFilePath, string id, VersionRang
                     PackagesPath = packagesPath,
                     FallbackFolders = fallbackFolders,
                     Sources = sources,
+                    OriginalTargetFrameworks = {
+                        framework.GetShortFolderName()
```

Comment:
don't call GetShortFolderName twice, it's not terribly expensive, but not cheap either.

Reply:
@rohit21agrawal good point, nice catch.